src/validation.rs

```diff
@@ -349,6 +349,11 @@ static DARWIN_ALLOWED_DYLIBS: Lazy<Vec<MachOAllowedDylib>> = Lazy::new(|| {
                 max_compatibility_version: "1.0.0".try_into().unwrap(),
                 required: true,
             },
+            MachOAllowedDylib {
+                name: "/System/Library/Frameworks/UniformTypeIdentifiers.framework/Versions/A/UniformTypeIdentifiers".to_string(),
+                max_compatibility_version: "1.0.0".try_into().unwrap(),
```

Comment:
I don't know how to determine this version.

Reply:
Run `otool -L` on the binary requiring this library. This prints the compatibility version that is embedded in the Mach-O library dependency data.